sqlmodel/_compat.py

```diff
@@ -335,6 +327,46 @@ def sqlmodel_validate(
                     setattr(new_obj, key, value)
         return new_obj
 
+    def sqlmodel_validate_python(
```

Comment:
Is it me or is this redefined later in this file?

Reply:
It is wrapped inside an if else block for pydantic v1 vs v2 at line 72.